homeassistant/components/onvif/event.py

```diff
@@ -156,7 +156,16 @@ async def async_parse_messages(self, messages) -> None:
             if not msg.Topic:
                 continue
 
-            topic = msg.Topic._value_1
+            # Topic may look like the following
+            #
+            # tns1:RuleEngine/CellMotionDetector/Motion//.
+            # tns1:RuleEngine/CellMotionDetector/Motion
+            # tns1:RuleEngine/CellMotionDetector/Motion/
+            #
+            # Our parser expects the topic to be
+            # tns1:RuleEngine/CellMotionDetector/Motion
+            topic = msg.Topic._value_1.rstrip(".").rstrip("/")
```

Comment:
rstrip will strip any of the listed characters in the string, it doesn't string match. So this should work too:
```suggestion
            topic = msg.Topic._value_1.rstrip("/.")
```

Reply:
Nice. Way better than the complex regex I started with 😉